.gitignore

```diff
@@ -1,5 +1,6 @@
 \#*#
 .#*
+*.swp
```

Comment:
I added this because I used Vim while working on the pull request. You are right that this line has nothing to do with a proposed change and I will remove it.